dm/loader/lightning.go

```diff
@@ -204,39 +207,30 @@ func (l *LightningLoader) restore(ctx context.Context) error {
 			}
 		}
 		cfg := lcfg.NewConfig()
-		if err = cfg.LoadFromGlobal(l.lightningConfig); err != nil {
+		if err = cfg.LoadFromGlobal(l.lightningGlobalConfig); err != nil {
 			return err
 		}
 		cfg.Routes = l.cfg.RouteRules
 		cfg.Checkpoint.Driver = lcfg.CheckpointDriverMySQL
 		cfg.Checkpoint.Schema = config.TiDBLightningCheckpointPrefix + dbutil.TableName(l.workerName, l.cfg.Name)
 		cfg.Checkpoint.KeepAfterSuccess = lcfg.CheckpointOrigin
-		param := common.MySQLConnectParam{
-			Host:             cfg.TiDB.Host,
-			Port:             cfg.TiDB.Port,
-			User:             cfg.TiDB.User,
-			Password:         cfg.TiDB.Psw,
-			SQLMode:          mysql.DefaultSQLMode,
-			MaxAllowedPacket: 64 * units.MiB,
-			TLS:              cfg.TiDB.TLS,
-		}
-		cfg.Checkpoint.DSN = param.ToDSN()
 		cfg.TiDB.Vars = make(map[string]string)
 		if l.cfg.To.Session != nil {
 			for k, v := range l.cfg.To.Session {
 				cfg.TiDB.Vars[k] = v
 			}
 		}
-
 		cfg.TiDB.StrSQLMode = l.cfg.LoaderConfig.SQLMode
-		cfg.TiDB.Vars = map[string]string{
-			"time_zone": l.timeZone,
-		}
-		if err = cfg.Adjust(ctx); err != nil {
```

Comment:
lightning will auto adjust config in [hehe](https://github.com/pingcap/tidb/blob/d83ee8cfdad434d212d3229c0be24d9d81ef7ab3/br/pkg/lightning/lightning.go#L192), don't need to adjust again

Reply:
hehe 😄